manuscript/manuscript.Rmd

```diff
@@ -541,13 +542,18 @@ Empirically, we estimate $h$ by
 
 where $N$ is the number of records in the dataset.
 
+Note that this proposed plot can be calculated over the dataset or over a
+hypothetical dataset in which the value of $x_i^{j}$ is set equal to $z$ for
+each record.
```

Comment:
For a grid of z values and averaging the model prediction at each z?

Reply:
That was my thought? I guess I didn't 100% follow how your revised PDP was generated, so feel free to mess with this language!